test/e2e/positive.go

```diff
@@ -286,14 +336,15 @@ func getNodeResourceTopology(tc *topologyclientset.Clientset, namespace, name st
 	return nrt
 }
 
-func deploy() error {
+func deploy(extraArgs []string) error {
```

Comment:
this is too low level. This function (and `remove`) should take an argument to distinguish between NFD and RTE and create the command line accordingly, avoiding to accept random data from the callers.